content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -1118,3 +1118,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 
 * The [deprecation policy](/docs/reference/using-api/deprecation-policy/) for Kubernetes explains
   the project's approach to removing features and components.
+* Starting in Kubernetes 1.24, new beta APIs are not enabled by default.  When enabling a beta
```

Comment:
I would target this PR to the branch _dev-1.24_ and then change the text to
```suggestion
* Since Kubernetes 1.24, new beta APIs are not enabled by default.  When enabling a beta
```